surveyor/jetstream.go

```diff
@@ -82,6 +82,11 @@ var (
 		Help: "Advisories about JetStream Consumer Delivery Exceeded events",
 	}, []string{"account", "stream", "consumer"})
 
+	jsDeliveryTerminatedCtr = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: prometheus.BuildFQName("nats", "survey", "jetstream_delivery_terminated_count"),
+		Help: "Advisories about JetStream Consumer Delivery Terminated events",
+	}, []string{"account", "stream", "consumer"})
```

Comment:
I just noticed this...   Should the JS labels mirror the `Statz` labels?

{"server_cluster", "server_name", "server_id", "account", "stream", "consumer"}